### ISSUE TITLE
Readme's app bootstrap no longer work with new app.rb dynamically pulling in version

### ISSUE THREAD

**Issue by Obversity** [katalyst/koi#289]
@pvawser01  Yesterday I noticed that the readme hadn't been updated and the setup instructions (`rails new ... `) were still referencing an old tag/branch. So without thinking too much, I updated the readme and pushed. Then I tested the new instructions from github, and they failed. 

![image](https://cloud.githubusercontent.com/assets/7999019/23693252/80624ee6-0423-11e7-9152-44ca94cb1abd.png)

It looks like the `require_relative` is trying to grab the version file from the wrong spot. I tried a couple of things that I thought might work, including building the require path myself with `__FILE__` etc but the issue persisted. 

We're going to need a solution to this if we're going to keep the version being dynamic. 

If it ends up being too hard, then perhaps we should just have a "version upgrade checklist" somewhere, so when we go to bump Koi up a version, we can make sure we change all the right spots:

- the version.rb file
- the readme bootstrap instructions
- the app.rb koi gem installation
- add any necessary things to the Upgrade.md
- Add changes to the release

**Comment by Obversity** [comment]
@pvawser01  I seem to remember you saying this was fixed, Pete. Can this issue be closed?

**Comment by pvawser01** [comment]
I have updated the read me with a tag that needs to be replaced before running the copied command. The only dynamic way would be by using pre commit hooks. Not sure it warrants that so this will suffice. Also, I have defaulted the gem file to use the dynamic koi version not the local version. Keep in mind this is an open source project. Just good practice. If you want to run it using your local koi, change it but put it back before committing.
